python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3292,39 +3292,49 @@ def test_stop_session(self):
             self.spark.conf.get("some.conf")
         self._check_no_active_session_error(e.exception)
 
-    def test_error_stack_trace(self):
-        with self.sql_conf({"spark.sql.pyspark.jvmStacktrace.enabled": True}):
```

Comment:
Doesn't this effectively change the API? What is the expectation when the user sets the SQL conf and moves from py4j to spark connect and back? Shouldn't we retain the option?

Reply:
`spark.sql.pyspark.jvmStacktrace.enabled` only used by spark connect python client and the option still works